code/datums/gamemodes/revolution.dm

```diff
@@ -121,6 +124,10 @@
 	if (world.time > win_check_freq)
 		win_check_freq += win_check_freq
 		check_win()
+	if (TIME >= waittime && !waittimed)
```

Comment:
This will include pre-game time as `TIME` is counted from server start time, not round start time:
![image](https://github.com/goonstation/goonstation/assets/20713227/befc1a2d-a1f3-4b45-9b28-69cb86d24ce6)
You'll need to store the round start time (unless it's already tracked somewhere?) and subtract it here if you want it to be accurate.

Reply:
it's tracked somewhere uhhhhhhhhhhh

Reply:
stat panel